app/src/main/java/com/arygm/quickfix/ui/Activity.kt

```diff
@@ -1,36 +1,81 @@
 package com.arygm.quickfix.ui
 
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.ui.navigation.BottomNavigationMenu
 import com.arygm.quickfix.ui.navigation.NavigationActions
 import com.arygm.quickfix.ui.navigation.Route
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun ActivityScreen(navigationActions: NavigationActions, isUser: Boolean) {
-  Scaffold(
-      topBar = { TopAppBar(title = { Text(text = "ACTIVITY Screen") }) },
-      bottomBar = {
-        // Boolean isUser = true for this HomeScreen
-        MeowBottomNavigationMenu(
-            selectedItem = Route.ACTIVITY, // Start with the "Home" route
-            onTabSelect = { selectedDestination ->
-              // Use this block to navigate based on the selected tab
-              navigationActions.navigateTo(selectedDestination)
-            },
-            isUser = isUser // Assuming the user is of type User
+fun ActivityScreen(navigationActions: NavigationActions, isUser: Boolean, LoD: Boolean) {
+
+    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
+
+    // Use Scaffold for the layout structure
+    Scaffold(
+        containerColor = backgroundColor,
+        topBar = {
+            Surface(
+                shape = RoundedCornerShape(bottomStart = 16.dp, bottomEnd = 16.dp),
+                modifier = Modifier.fillMaxWidth()
+            ){
+                TopAppBar(
+                    title = {
+                        Box(modifier = Modifier.fillMaxWidth()) {
+                            Text(
+                                text = "Activity",
+                                color = backgroundColor,
```

Comment:
One possible solution to avoid repeating the scaffold in different screens is to have only one for your whole app for example in QuickFixApp.kt that conditionally shows the topAppBar and BottomBar

Pseudo code
```
Scaffold(
        bottomBar = {
            if (shouldShowBottomBar) {
                QuickFixBottomNavBar()
            }
        },
        topBar = {
            if (shouldShowTopBar) {
                QuickFixTopAppBar()
            }
        }
) { scaffoldPadding ->
        QuickFixNavHost()
}
```

To get the values of shouldShow... you could have a navigation class that specifies such things for each screen you have (it would essentially function as a sort of dict). Another way to handle this is with an AppState object like in this [example](https://github.com/jshvarts/ComposeConditionalBottomNav). Keep in mind this is only a suggestion.